internal/kallsyms/kallsyms_test.go

```diff
@@ -7,18 +7,21 @@ import (
 	"github.com/go-quicktest/qt"
 )
 
+var kallsyms = []byte(`0000000000000000 t hid_generic_probe	[hid_generic]
+A0000000000000EA t writenote
+00000000000000A0 T tcp_connect
+00000000000000B0 B empty_zero_page
+00000000000000C0 D kimage_vaddr
+00000000000000D0 R __start_pci_fixups_early
+00000000000000E0 V hv_root_partition
+00000000000000F0 W calibrate_delay_is_known
+A0000000000000AA a nft_counter_seq	[nft_counter]
+A0000000000000BA b bootconfig_found
+A0000000000000CA d __func__.10
+A0000000000000DA r __ksymtab_LZ4_decompress_fast
+A0000000000000EA t writenote`)
```

Comment:
Nit: Is it typical for duplicate symbols to have the same address? Typically, they come from separate compilation units, so their addresses shoud differ. Not important for the sake of the test, but maybe we could tolerate duplicates as long as their addresses match.